migrations/versions/76c949b1235f_.py

```diff
@@ -68,7 +69,8 @@ def downgrade():
     op.alter_column('events_version', 'can_pay_by_paypal', new_column_name='pay_by_paypal')
     op.alter_column('events_version', 'can_pay_by_cheque', new_column_name='pay_by_cheque')
     op.alter_column('events_version', 'can_pay_onsite', new_column_name='pay_onsite')
-    op.alter_column('events_version', 'has_sessions_speakers', new_column_name='has_session_speakers')
+    op.alter_column('events_version', 'has_sessions_speakers',
```

Comment:
@SaptakS 
This should have been 
`op.alter_column('events', 'is_sessions_speakers_enabled', new_column_name=has_sessions_speakers')`
made a fix in [PR#3846](https://github.com/fossasia/open-event-orga-server/pull/3846)